go.mod

```diff
@@ -1,9 +1,14 @@
 module github.com/graph-gophers/graphql-go
 
-go 1.13
+go 1.18
 
 require (
 	github.com/opentracing/opentracing-go v1.2.0
 	go.opentelemetry.io/otel v1.6.3
 	go.opentelemetry.io/otel/trace v1.6.3
 )
+
+require (
+	github.com/go-logr/logr v1.2.3 // indirect
+	github.com/go-logr/stdr v1.2.2 // indirect
```

Comment:
Why do we need these dependencies for?

Reply:
https://go.dev/doc/go1.17#go-command This change in 1.17 is the likely cause of this change to go.mod